pkg/jq/jq.go

```diff
@@ -14,6 +14,12 @@ import (
 
 // Evaluate a jq expression against an input and write it to an output.
 func Evaluate(input io.Reader, output io.Writer, expr string) error {
+	return EvaluateFormatted(input, output, expr, false)
+}
+
+// Evaluate a jq expression against an input and write it to an output, either without
+// whitespace or with 2 spaces of indentation per level (pretty printed).
```

Comment:
Forgot to add a request: the code docs of the original Evaluate function didn't specify in which cases this prints JSON payloads and in which raw values, but maybe your change is a good time to add this info, if you'd be willing?

Basically, this function takes a jq expression to extract and print raw (i.e. non-JSON) values to an output writer. However, if the selected node is not a scalar, but is instead a JSON object or an array, this function will print the resulting JSON. This is a hack meant not really to be consumed by scripts directly, but to offer feedback to the person crafting the jq expression, prompting them to potentially drill deeper to isolate all the scalar values.

Reply:
Sure, no problem. I had assumed that the code was designed to act as if `jq --raw` was being used (which does exactly this, output scalar values directly and not as JSON scalars).